Backend/DataAccessLayer/Repositories/PointRepository.cs

```diff
@@ -13,7 +13,7 @@ private bool ValidateLongLat(Point point)
         {
             if (point.Longitude > 180 || point.Longitude < -180 ||
                 point.Latitude > 90 || point.Latitude < -90)
-                throw new ArgumentOutOfRangeException("Longitude/Latitude value out of range.");
+                throw new ArgumentOutOfRangeException();

```

Comment:
No longer need argument, as this is caught and turned into a Point specific exception in the next layer.